.travis.yml

```diff
@@ -1,3 +1,7 @@
+git:
+  depth: 25
```

Comment:
why do we need this

Reply:
An error was raised by travis indicating that it could n

> This is good work, You could center the contents on the left.

Let me fix that @Philipotieno ,
Thanks

Reply:
After force pushing several times, Travis threw an error indicating that it could not identify the depth and this is the solution I found(Increasing the depth of possible force pushed commits)